lib/screens/topic_screen.dart

```diff
@@ -12,22 +14,23 @@ class TopicScreen extends StatefulWidget {
 
 class _TopicScreenState extends State<TopicScreen> {
 
-  List<Tuple7<String, String, String, String, String, String, int >> _articles;
+  List<Article> _articles;
+  bool _isLoading = true;
 
   void _initTopic() async {
-    new ArticleRepo().getArticle('lion').then((_articles) async{
-      if(_articles.text.isNotEmpty){
-        print(_articles.text);
-
-      }
+    new ArticleRepo().getArticle('lion').then((articles) async{
+      _articles = articles;
     });
   }
 
   @override
     void initState() {
       // TODO: implement initState
       super.initState();
-      _initTopic();
+      setState(() {
```

Comment:
You should not wrap the complete method _initTopic in setState.
Inside _initTopic, you can have:
```
    new ArticleRepo().getArticle('lion').then((articles) async{
       setState(() {
         _articles = articles;
        });
```

Reply:
Yeah, I did those changes sir, but went to meeting so was not able to commit it.